model/tacotron2.py

```diff
@@ -27,6 +27,7 @@ def __init__(
         num_speakers: int = 1,
         controls: bool = False,
         controls_dim: int = 0,
+        description_tokens: bool = False,
```

Comment:
I suspect part of the reason you aren't encountering problems is that the `description_tokens` parameter is set to False by default. This is correct for when this code is eventually merged, but since no Tacotron models are currently configured to activate this parameter (unless one was added without committing it), training is being run without any of this new code being executed.

For the time being, change the default value to `True` and see what happens. We can change it back before merging this code.